lua/packer.lua

```diff
@@ -178,8 +180,17 @@ packer.install = function(...)
       await(a.main)
       plugin_utils.update_helptags(install_paths)
       plugin_utils.update_rplugins()
-      local delta = string.gsub(vim.fn.reltimestr(vim.fn.reltime(start_time)), ' ', '')
-      display_win:final_results(results, delta)
+
+      if not vim.tbl_isempty(results.rocks) then
+        await(neorocks.install_rocks(results.rocks))
+      else
+        log.info("No rocks to install")
```

Comment:
Nitpick: I think the "no rocks" behavior should be silence, rather than logging. That said, totally makes sense for debugging WIP.

Reply:
Added TODO